flow/model/qvalue/avro_converter.go

```diff
@@ -96,6 +98,30 @@ type QValueAvroConverter struct {
 	Nullable  bool
 }
 
+func HStoreToJSON(hstore string) (string, error) {
+	re := regexp.MustCompile(`"(.*?)"=>(?:"([^"]*)"|NULL)`)
+	matches := re.FindAllStringSubmatch(hstore, -1)
```

Comment:
Don't use regex for this, reference the postgres hstore parser

https://github.com/postgres/postgres/blob/bea18b1c949145ba2ca79d4765dba3cc9494a480/contrib/hstore/hstore_io.c#L476

Granted, the hstore_out will always have quotes, but you need to process `\\` & `\"`